<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ClientOrder</class>
 <widget class="QDialog" name="ClientOrder">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1263</width>
    <height>805</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <widget class="QFrame" name="frame_title_Report">
     <property name="maximumSize">
      <size>
       <width>274</width>
       <height>84</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="ImgReport">
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap resource="Recource.qrc">:/rec/img/report.png</pixmap>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lbReport">
        <property name="minimumSize">
         <size>
          <width>184</width>
          <height>64</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 20pt &quot;Segoe UI&quot;;</string>
        </property>
        <property name="text">
         <string>Отчет Заказов</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="1" rowspan="3">
    <widget class="QTableWidget" name="twOrg">
     <property name="minimumSize">
      <size>
       <width>561</width>
       <height>561</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QFrame" name="frame">
     <property name="minimumSize">
      <size>
       <width>335</width>
       <height>326</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>492</width>
       <height>464</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="lbNumber">
        <property name="text">
         <string>Номер заказа </string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="leOrder_number"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="lbDate">
        <property name="text">
         <string>Дата Заказа</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="leDate"/>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="lbID_c">
        <property name="text">
         <string>ФИО Клиента </string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="leFio_client"/>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="lbPay">
        <property name="text">
         <string>Оплата</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLineEdit" name="lePay"/>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="lbSum">
        <property name="text">
         <string>Сумма Заказа</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QLineEdit" name="leSum"/>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QFrame" name="frame_4">
     <property name="minimumSize">
      <size>
       <width>351</width>
       <height>41</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>351</width>
       <height>41</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QPushButton" name="btnBack">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>111</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>Назад </string>
      </property>
     </widget>
    </widget>
   </item>
   <item row="3" column="0" colspan="2">
    <widget class="QTextEdit" name="teResult">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>101</height>
      </size>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="Recource.qrc"/>
 </resources>
 <connections/>
</ui>
